travis/shared_configs/docs-build.yml

```diff
@@ -1,18 +1,46 @@
+version: ~> 1.0
+
 jobs:
   include:
     - stage: test
       name: "Docs Build"
-      install:
+      workspaces:
+        create:
+          name: docs
+          paths:
+            - docs/build/
+        use: conda
+      install: skip
+      before_script:
+        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+        - bash miniconda.sh -b -p $HOME/miniconda
+        - export PATH="$HOME/miniconda/bin:$PATH"
+        - conda config --set always_yes yes --set changeps1 no
+        - conda config --remove channels defaults
+        - conda config --add channels conda-forge
+        - conda config --add channels "file://`pwd`/bld-dir"
+        # Useful for debugging
+        - conda info -a
+        - echo "Conda Environment Name':' ${CONDA_ENV_NAME:=testenv}"
+        # Manage conda environment
+        - conda create -n ${CONDA_ENV_NAME} python=$PYTHON_VERSION ${CONDA_PACKAGE} ${CONDA_EXTRAS}
+        - source activate ${CONDA_ENV_NAME}
+        # Useful for debugging
+        - conda list
+
```

Comment:
Why is conda necessary for the docs-build?

Reply:
It makes it easier to install dependencies since most of our internal packages are damaged at PyPI. E.g. Happi is broken and can't be installed. After we have a uniform and standard procedure for the uploads and packages we can switch the docs back to be just pip and remove conda.